pymodbus/framer/base.py

```diff
@@ -60,6 +60,21 @@ def _validate_slave_id(self, slaves: list, single: bool) -> bool:
             return True
         return self._header["uid"] in slaves
 
+    async def asyncProcess(self, exec_request):
```

Comment:
As written earlier this breaks the architecture layer. However it is not a no-go.